core/src/test/scala/org/genivi/sota/core/client/FakeReposerverClient.scala

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright: Copyright (C) 2016, Jaguar Land Rover
+ * License: MPL-2.0
+ */
+package org.genivi.sota.core.client
+
+import akka.http.scaladsl.model.Uri
+import akka.http.scaladsl.util.FastFuture
+import com.advancedtelematic.libats.data.Namespace
+import com.advancedtelematic.libtuf.data.TufDataType.{Checksum, RepoId}
+import com.advancedtelematic.libtuf.reposerver._
+import java.util.UUID
+import scala.concurrent.Future
+
+object FakeReposerverClient extends ReposerverClient {
+
+  private var store: Map[Namespace, RepoId] = Map.empty
```

Comment:
We often use `java.util.concurrent.ConcurrentHashMap` in these kind of tests, since in the future more tests might need this and we can get subtle race conditions.